components/intro-template.tsx

```diff
@@ -0,0 +1,139 @@
+                Create and edit content
+              </div>
+              <div>
+                Go to
+                <LinkAttribute href="/studio" text="Sanity Studio" blue />
+              </div>
+              <div className="text-xs text-gray-700">
+                We embedded it in this project under
+                <LinkAttribute
+                  href="https://cms-sanity-2-eight.vercel.app/studio"
+                  text="https://cms-sanity-2-eight.vercel.app/studio"
+                />
+              </div>
+              <div className="mt-2">
+                Check out instant previews:
+                <LinkAttribute href="/studio" text="Create a document" blue />
+              </div>
+            </div>
+          }
+        />
+
+        <Box
+          circleTitle="2"
+          element={
+            <div>
+              <div className="col-span-2 mb-2 font-bold">
+                Modify and deploy the project
+              </div>
+              <div className="mb-2">
+                Clone the
+                <LinkAttribute
+                  href={`https://${process.env.NEXT_PUBLIC_VERCEL_GIT_PROVIDER}.com/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_OWNER}/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_SLUG}`}
+                  text="repo"
+                  blue
+                />
+                for this project
+              </div>
+              <div className="mb-2">
+                Learn how to configure your project locally, make changes and
+                deploy them:
+                <LinkAttribute
+                  href={`https://${process.env.NEXT_PUBLIC_VERCEL_GIT_PROVIDER}.com/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_OWNER}/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_SLUG}/blob/main/README.md`}
+                  text="Read the README"
+                  blue
+                />
+              </div>
+              <div className="mb-2">
+                Check out
+                <LinkAttribute
+                  href="https://www.sanity.io/docs"
+                  text="Sanity Docs"
+                  blue
+                />
+              </div>
+              <div className="mt-2">
+                Join
+                <LinkAttribute
+                  href="https://slack.sanity.io/"
+                  text="Sanity Slack Community"
+                  blue
+                />
+                to get help and ask questions
+              </div>
+            </div>
+          }
+        />
+      </div>
+    </div>
+  )
+}
+
+function Box({
+  circleTitle,
+  element,
+}: {
+  circleTitle: string
+  element: JSX.Element
+}) {
+  return (
+    <div className="grid grid-flow-col grid-rows-1 place-content-start gap-3">
+      <div className="row-span-3">
+        <div className="relative flex h-6 w-6 items-center justify-center rounded-full bg-gray-200 p-4 text-center">
+          {circleTitle}
+        </div>
+      </div>
+      {element}
+    </div>
+  )
+}
+
+function LinkAttribute({
```

Comment:
Why use `LinkAttribute` and the `Link` from `next` here? 🤔

Reply:
I understand,

maybe I'm nitpicking, but my question remains why we're using the Link from the next package on [line 129](https://github.com/sanity-io/vercel-next.js-examples-cms-sanity-studio-v3/pull/23/files#diff-e9d19edfef4c7cfe3cef2d8737460ea41fc6f86660ff3c87bf5173afa4ef5395R129) instead of just the `a`

Reply:
Ahh, I thought I could use the next link for external links as well, but after reading up on it I understand that this is not best practice! I fixed it now 🙌